go.mod

```diff
@@ -1,41 +1,40 @@
 module github.com/aws/amazon-sagemaker-operator-for-k8s
 
-go 1.12
+go 1.13
 
 require (
 	cloud.google.com/go v0.38.0 // indirect
 	github.com/Jeffail/gabs/v2 v2.0.0
 	github.com/adammck/venv v0.0.0-20160819025605-8a9c907a37d3
-	github.com/aws/aws-sdk-go v1.25.36
-	github.com/aws/aws-sdk-go-v2 v0.19.0
+	github.com/aws/aws-sdk-go v1.30.0 // indirect
```

Comment:
Is this dependency automatically ?

Reply:
Yes. So I removed our direct dependency on this, but something else re-introduces it as an indirect dependency (I believe it is v2 that brings it back). Either way we shouldn't need to worry about managing this anymore.

Reply:
Nope. Gone now. Just needed a `go mod tidy`